### DISCUSSION TITLE
农场内的分流指数流速系数存在严重问题，会引起农场系统崩乱，建议更正！

### DISCUSSION THREAD

**Discussion by yancanlou** [famland/website#91] [Ideas]
白皮书中说到：
**农场的分流指数即多少由其平均共享值确定，但又指出创建成功后，农场内可自由移除或自主退出的，Famland数量即便低于30个，农场成员基本田提取做市资金，低于500U也不影响农场正常工作。**

### 按现有算法，这里面有极大的问题：
**1.低于30个不受影响？**
没有指出低于多少，如果是只留1个，那么这块田的产出是全系统最大的。
因为这1块田的平均共享值就是它本身。而平均共享值做为分流指数占比参与总流速分配，其将得到最大流速。
**2.农场成员基本田提取做市资金，低于500U也不影响？**
这样在建场后同样可以撤出部分资金，不断建新场，对生态也是一种严重破坏。
**3.这是最大的不公了！**
那么接下来，将出现大户疯狂的建，撤刷YL,人心流失而系统崩坏。
**4.好了，如果上述两种情况都受控后，是不是就没有问题了呢？**
不是，即便受限于30块，如此30块将是最优农场数量，不存在增加的可能，增加不是最优收益比。
### 如何改进呢？？？
分析白皮书：
**子农场分流指数=子农场总共享值/子农场Famland总数量x流速系数**
即：
流速系数初始值为=1，每增加30块，系数增加万分之1。
表达为
流速系数=1+（子农场Famland总数量/30x0.0001）
例如农场为60块则：
1+60/30x0.0001=1.0002    ？？？够吃饭吗？

应当更正为：
流速系数为，系数参数f与子农场Famland总数量/30倍数之积
数学表达为：
**流速系数=子农场Famland总数量/30*f，f取1**
例如农场为60块则：
60/30x1=2    ！！！多建没影响产能。
则
### 补救建议：
**子农场分流指数=子农场总共享值/子农场Famland总数量x流速系数**
可结合上式
子农场分流指数可以简化为
### （现况农场未上线）如果撤出做市资金目前不会降低其共享值，则农场内的基本田不能撤出做市资金，只能在原生农场内撤出。
如此就能解决上述问题了，请参考！

**Comment by yancanlou** [comment]
设想补充:
子农场分流指数=子农场总共享值/30*f 
如果f取1，则30这个建场基数就变成分流指数的系数。
如果f不取1，可以考率取子农场内的差比值
**f=子农场最低共享值/子农场最高共享值**
那么
**子农场分流指数=（子农场总共享值/30）x（子农场最低共享值/子农场最高共享值）**
或按白皮书定义表达
**子农场分流指数=子农场总数/30x子农场平均共享值x子农场最低共享值/子农场最高共享值**

由此式可知，如果引入农场内的差比值，将影响场内成员对于最低共享值成员和最高共享值的排斥，因为拉低了系数。
**最低共享值差太多农场平均值，可以淘汰出去找到其它农场相合适的，这样有利于促进共享值的提升；
最高共享值偏离太多农场平均值，其实不用等团队去请他高就，他自己也会找到合适的农场发展；
有了这样环境，每个农场都会出现一段时交错的人员流动，不断的配比，促进了生态发展。**

### 最终只有相近的共享值是不够，团队也必须要有相同的进步曲线的Famland才能结成最优共识，获得最佳收益比。

**Comment by otto-001** [comment]
哈哈，这个标题让人有点紧张。别系统没崩溃，大家信心先崩溃了。😁
现在试运营阶段，别把气氛搞这么紧张。😉
任何机制都是不断完善的，我得像你学习，认真研究白皮书并提出合理化建议😁

**Reply by yancanlou** [reply]
这个问题，本来就是很严重啊，事关共享理念的核心

**Comment by linshande** [comment]
这个问题刚提出，马上就有人开始建最小号田了

**Reply by otto-001** [reply]
是不是看到这个受到启发了😂😂
